MiguelM.txt

```diff
@@ -1,39 +1,39 @@
-//Hey Lambda! My name is Miguel Marquez and this is my introduction
+//Hey Lambda! My name is Miguel Marquez and this is my "alternate reality"
```

Comment:
Good work! When committing try to be descriptive as to what you changed versus Final1..2..3  This will help you down the road when you have many commits and want to see where you made certain changes.